Angela/pcx 3093 update logs examples for consistency (#3131)
cloudflare/cloudflare-docs: products/logs/src/content/reference/logpush-api-configuration/index.md

The table below summarizes the job operations available. All the examples in this page are for zone-scoped data sets. Account-scoped data sets should use `/accounts/<ACCOUNT_ID>` instead of `/zone/<ZONE_ID>`. For more information, refer to the [Log fields](/reference/log-fields) page.

The `<ZONE_ID>` argument is the zone id (hexadecimal string). The `<ACCOUNT_ID>` argument is the organization id (hexadecimal string). These arguments can be found using [API's zones endpoint](https://api.cloudflare.com/#getting-started-resource-ids).
The `<JOB>` argument is the numeric job id. The `<DATASET>` argument indicates the log category (such as `http_requests`, `spectrum_events`, `firewall_events`, `nel_reports`, or `dns_logs`).

<TableWrap>

| Operation | Description | URL |
|---|---|---|

> | POST | Create job | <em>https://api.cloudflare.com/client/v4/zones/&lt;zone_id&gt;/logpush/jobs</em> |

Reviewer: You should capitalize the "zone_id", "job", and "dataset" placeholders in all the table rows (URL column).

```bash

> $ curl -s -XPOST https://api.cloudflare.com/client/v4/zones/<ZONE_ID>/logpush/ownership \

Reviewer: ```suggestion
$ curl -s -X POST https://api.cloudflare.com/client/v4/zones/<ZONE_ID>/logpush/ownership \
```

The four options that you can customize are:

> 1. Fields: See *[Log fields](/reference/log-fields/)* for the currently available fields. The list of fields is also accessible directly from the API: `https://api.cloudflare.com/client/v4/zones/<ZONE_ID>/logpush/datasets/<DATASET>/fields`. Default fields: `https://api.cloudflare.com/client/v4/zones/<ZONE_ID>/logpush/datasets/<DATASET>/fields/default`.

Reviewer: ```suggestion
1. Fields: Refer to [Log fields](/reference/log-fields/) for the currently available fields. The list of fields is also accessible directly from the API: `https://api.cloudflare.com/client/v4/zones/<ZONE_ID>/logpush/datasets/<DATASET>/fields`. Default fields: `https://api.cloudflare.com/client/v4/zones/<ZONE_ID>/logpush/datasets/<DATASET>/fields/default`.
```

> 1. Sampling rate: Value can range from 0.001 to 1.0 (inclusive). `sample=0.1` means return 10% (1 in 10) of all records.

Reviewer: ```suggestion
1. Sampling rate: Value can range from 0.001 to 1.0 (inclusive). `sample=0.1` means `return 10% (1 in 10) of all records`.
```

> 1. Timestamp format: The format in which timestamp fields will be returned. Value options: unixnano (default), unix, rfc3339.

Reviewer: ```suggestion
1. Timestamp format: The format in which timestamp fields will be returned. Value options: `unixnano` (default), `unix`, `rfc3339`.
```
